jbpm-audit/src/main/java/org/jbpm/process/audit/NodeInstanceLog.java

```diff
@@ -35,7 +35,8 @@
 @Entity
 @Table(name = "NodeInstanceLog", indexes = {@Index(name = "IDX_NInstLog_pInstId", columnList = "processInstanceId"),
                                         @Index(name = "IDX_NInstLog_nodeType", columnList = "nodeType"),
-                                        @Index(name = "IDX_NInstLog_pId", columnList = "processId")})
+                                        @Index(name = "IDX_NInstLog_pId", columnList = "processId"),
+                                        @Index(name = "IDX_NInstLog_workItemId", columnList = "workItemId")})
```

Comment:
There's a discrepancy in naming: here it's "IDX_NInstLog...", but in most of the files (all but h2) is "NodeInstanceLog...", please unify them

Reply:
onde